packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -216,6 +216,94 @@ export interface ClusterOptions {
    * @default true
    */
   readonly outputConfigCommand?: boolean;
+
+  /**
+   * Configure access to the Kubernetes API server endpoint.
+   * This feature is only available for kubectl enabled clusters, i.e `kubectlEnabled: true`.
+   *
+   * @see https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html
+   *
+   * @default EndpointAccess.publicAndPrivate()
+   */
+  readonly endpointAccess?: EndpointAccess;
+
+  /**
+   * Environment for the kubectl execution. Only relevant for kubectl enabled clusters.
+   *
+   * These variables will be injected to the handler executing kubectl/helm commands.
+   *
+   * @default - No environment.
+   */
+  readonly kubectlEnvironment?: { [key: string]: string }
```

Comment:
How come eslint didn't yell here?

```suggestion
  readonly kubectlEnvironment?: { [key: string]: string };
```

Reply:
Apparently they are not required if there is a preceding curly brace that closes the block: https://eslint.org/docs/rules/semi#omitlastinonelineblock. 

Couldn't figure out if this is actually the default behavior, but i guess it is since I verified it does complain if i just remove a semi-colon somewhere without a curly brace.